### ISSUE TITLE
Add selectionBackgroundColor property

### ISSUE THREAD

**Issue by andris-zalitis** [enfluid/apple-scrub-bar#6]

**Comment by rudolf-adamkovic** [comment]
Maybe `selectedBackgroundColor` would be a better name to match Cocoa conventions?

**Comment by andris-zalitis** [comment]
So in Cocoa we have `selectedBackgroundView` for `UITableViewCell` and `UICollectionViewCell` which is "The view that is displayed just above the background view when the cell is selected.".

So although `selectedBackgroundColor` fits convention, I think it would work if the whole `SlidingSegmentedControl` would be the one that gets selected, but it's not. Selected background color will be used for two purposes - to show the selected segment and to show the scrubbing area which encompasses all segments while pan gesture is in progress.

I think `selectedSegmentBackgroundColor` would be perfect if we only used it for the currently selected segment. Since it is a bit more universal (and shows selection) I opted for `selectionBackgroundColor` but other suggestions are welcome.
